packages/frontend/src/modules/Auth/views/Login.tsx

```diff
@@ -39,9 +40,13 @@ export default function LoginPage() {
     return mutation.mutateAsync(data);
   };
 
+  const handleTitleClick = () => {
+    router.push('/login');
+  }
+
   return (
     <Dialog open>
-      <DialogTitle>Ease Trip</DialogTitle>
+      <DialogTitle onClick={handleTitleClick} style={{ cursor: 'pointer'}}>{LOGO_NAME}</DialogTitle>
```

Comment:
I feel like using Dialog in this page is quite weird. It doesn't seem to be a Dialog. And I feel it weird that users can click on the title. If you do want to make it clickable, you can
```
<DialogTitle>
  <EzTextButton onClick={xxx}>xxx</EzTextButton>
</DialogTitle>
```

Reply:
Let's refactor at the next sprint.
Bcz Recoil seems like this is a popup, not a page.